src/converters/lintConfigs/rules/ruleConverters/tests/adjacent-overload-signatures.test.ts

```diff
@@ -1,6 +1,8 @@
+import { describe, expect, test } from "@jest/globals";
+
 import { convertAdjacentOverloadSignatures } from "../adjacent-overload-signatures";
 
-describe(convertAdjacentOverloadSignatures, () => {
+describe("convertAdjacentOverloadSignatures", () => {
```

Comment:
https://github.com/facebook/jest/issues/12483

I've added a todo to write a lint rule for always or never using the stringy version to my todo.